docs/reference/edot-collector/components/elasticsearchexporter.md

```diff
@@ -131,32 +131,27 @@ The `elasticsearch.index` attribute is removed from the final document if it exi
 
 ## Performance and batching
 
-### Using sending queue
-
-The {{es}} exporter supports the `sending_queue` setting, which supports both queueing and batching.  The sending queue is deactivated by default.
-
-You can turn on the sending queue by setting `sending_queue::enabled` to `true`:
-
-```yaml subs=true
-exporters:
-  elasticsearch:
-    endpoint: https://elasticsearch:9200
-    sending_queue:
-      enabled: true
+### Queuing and batching
+
+The {{es}} exporter supports the common `sending_queue` settings, which enable both queuing and batching. The default sending queue is configured to do async batching with the following configuration:
+
+```yaml
+sending_queue:
+  enabled: true
+  num_consumers: runtime.NumCPU()
+  queue_size: <based on queue.mem.events>
+  block_on_overflow: true
+  wait_for_result: true
+  batch:
+    flush_timeout: 10s
+    min_size: 0
+    max_size: <based on flush::bytes>
+    sizer: items
 ```
 
-### Internal batching (default)
-
-By default, the exporter performs its own buffering and batching, as configured through the `flush` setting, unless the `sending_queue::batch` or the `batcher` settings are defined. In that case, batching is controlled by either of the two settings, depending on the version.
-
-### Custom batching
+The default configurations are chosen to be closer to the defaults with the exporter's previous built-in batching feature. For more details on the `sending_queue` settings, refer to the [`exporterhelper` documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md).
```

Comment:
i'm afraid these need to be in a applies_to 9.3

Reply:
All the default configs? If that's the case, we could put the PR on hold till 9.3 is up.